applications/tasks/src/tasks_maintenance.erl

```diff
@@ -110,13 +108,6 @@ start(TaskId) ->
         {'error', Reason} -> print_error(Reason)
     end.
 
--spec restart(text()) -> 'no_return'.
-restart(TaskId) ->
```

Comment:
I disagree with removing this. I find it quite useful when debugging tasks that failed to run because of errors in the code. Avoids having to create/start the job all over again, and I can rerun them many times as needed.